litinit.go

```diff
@@ -60,28 +57,28 @@ func litSetup(conf *config) *[32]byte {
 	// create home directory
 	_, err = os.Stat(preconf.LitHomeDir)
 	if err != nil {
-		fmt.Println("Error while creating a directory")
```

Comment:
The log file is not yet configured there. So using log and fmt both write to stdout (it will only write to stdout). Right? But it adds a package import.

Reply:
a package import vs a panic every time we print out the help command